### PR TITLE
Create cve.md

### PR DESCRIPTION
**Pull request by bstuder99**
Creates new doc on CVE protocol based on this issue
https://github.com/kubecost/docs/issues/425

### CODE REVIEW

**Comment by srpomeroy** [comment]
IMO this content should be added to our existing security doc.
https://github.com/kubecost/docs/blob/main/security.md

**Comment by bstuder99** [comment]
I'm on the fence on this. This is a pretty short article but the specifics mentioned in here feel a bit out of place in our existing security doc, which is mostly conveyed through short bullet points and doesn't link off-site anywhere like this doc will. I think it should be its own section in the security doc, probably right after Code Security.

File: cve.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# CVE Protocols
+
+Kubecost will never ask for or access your confidential data, and we try to ensure your information is always secure. Here is our proactive approach for ensuring common vulnerabilities and exposures (CVEs) are spotted and handled to ensure our product remains safe to use.
+
+Due to Kubecost’s multiple base dependencies, it’s important to track vulnerabilities in our product as well as those dependencies. We use [Aqua’s Trivy](https://github.com/aquasecurity/trivy) to scan our containers as well as any [Helm chart](https://github.com/kubecost/integration-ci-cd/blob/main/.github/workflows/nightly-scan-dependencies.yaml) dependencies nightly to track potential CVEs. The specific action used can be viewed [here](https://github.com/aquasecurity/trivy-action). Potential vulnerabilities will always be fixed in our upcoming release.
```
Comment (michaelmdresser) [inline cve.md:5]: The `[Helm chart]` link won't work for non-Kubecost folks because kubecost/integration-ci-cd is private.

File: cve.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+IWe recommend users ensure their version of Kubecost is always up to date. Keep up with our monthly releases [here](https://github.com/kubecost/cost-analyzer-helm-chart/releases).
```
Comment (michaelmdresser) [inline cve.md:7]:
```suggestion
We recommend users ensure their version of Kubecost is always up to date. Keep up with our monthly releases [here](https://github.com/kubecost/cost-analyzer-helm-chart/releases).
```

**Comment by michaelmdresser** [comment]
I'm in agreement with Sean. If anything, this is probably a subsection of Code Security.

**Comment by michaelmdresser** [comment]
From `security.md`: "The Engineering team monitors for patches that address issues or vulnerabilities in production systems. Patches are staged in the Staging environment and monitored to ensure stability and compatibility. After they pass monitoring requirements, the patches are rolled out to Production."

That essentially describes our CVE process, but doesn't specifically call out CVEs. That security doc may be part of our SOC2 process, so I'm not sure if the wording is particularly important.

I don't mind this being it's own doc either. Not a big deal IMO!

**Comment by srpomeroy** [comment]
The topic of vulnerability scanning typically comes up as part of a security review process and the rest of the items on the security page are also relevant to that end. 

That and we have enough of a docs sprawl issue. 😄

**Comment by bstuder99** [comment]
I'm rewriting the Security doc and it looks like this doc will be reduced to a single line

**Comment by bstuder99** [comment]
Content moved to https://github.com/kubecost/docs/pull/439

**Pull request closed without merging**